### ISSUE TITLE
About the MSE loss and CE loss

### ISSUE THREAD

**Issue by Lyan-ing** [facebookresearch/dino#205]
In your paper, the results with MSE loss only have 52.6 and 62.4 accuracies.  I'm not sure whether you fine-tune the hyper-parameters and train a  best model?

![图片](https://user-images.githubusercontent.com/65017880/174930321-cf7e0729-6776-4eea-9222-8a0710180606.png)

**Comment by woctezuma** [comment]
If I remember right, weights are frozen for `k-NN` (no finetuning), and only the classification head is finetuned for `Linear`.

**Comment by Lyan-ing** [comment]
thank you for your reply. Why the model trained with MSE loss is much worse than CE loss?

**Comment by woctezuma** [comment]
It is not explained in the paper, just observed.

![Paper](https://i.ibb.co/BLvkHwt/2022-06-24-09-48-10.png)

![Paper](https://i.ibb.co/B4d4QN5/2022-06-24-09-49-16.png)

**Comment by mathildecaron31** [comment]
Hi, this is indeed an empirical observation and don't have an explanation at this point.
